### DISCUSSION TITLE
How to more accurately track time spent in zoom/slack meetings?

### DISCUSSION THREAD

**Discussion by douglasg14b** [ActivityWatch/activitywatch#904] [Q&A]
Often when in zoom meetings I'm either sharing my screen, or in another application taking notes or doing some work.

It would be nice if Zoom was tracked whenever it was on an open meetings, regardless of the focused window. Is this possible?

How might a custom watcher keep track of this?

**Comment by 2e3s** [comment]
If you open Zoom only for meetings, a custom watcher could check and send data if there is a Zoom process running, of there is a certain window in the windows list, but it wouldn't affect other window activity stats.

**Reply by BelKed** [reply]
You can achieve a similar result by using the `Always count as active pattern` preference in [`Settings`](http://localhost:5600/#/settings):

<img width="525" alt="Screenshot 2023-07-09 at 09 39 23" src="https://github.com/ActivityWatch/activitywatch/assets/66956532/f8d92404-3601-4bde-a19d-9f026dbd641f">

**Reply by 2e3s** [reply]
> or in another application taking notes or doing some work

This would only work if you don't do anything on PC and the zoom's window is open.
